suclass/cms/templates/body-post.html

```diff
@@ -1 +1,14 @@
-<script id="ze-snippet" src="https://static.zdassets.com/ekr/snippet.js?key=8ca24832-80a4-4e66-ba71-c44c36c96b85"></script>
+<!-- 
+We have to wait to append this script to the page, otherwise there's a chance the zendesk widget does not show up on the page. Most likely, due to a race condition where other elements of the page must load first.
+-->
+<script>
+	$(document).ready(
```

Comment:
Let's make indents 2 spaces to match other files in the theme `template` directories.